api/views/course.py

```diff
@@ -195,7 +199,7 @@ def my_grades(self, request, pk):
                     "question_details": [
                         {
                             "title": uqj.question.title,
-                            "question_grade": uqj.tokens_received,
+                            "question_grade": uqj.formatted_current_tokens_received,
```

Comment:
i dont think you should use the formatted grade here since this is an API and can/will be used programmatically.
You have 2 options
1. add a new field `question_grade_formatted`.
2. Format the grade when writing to csv.